### PR TITLE
Update dependencies

### LINKED ISSUE #27: Update Dependencies
Update dependencies to latest version where possible and document any dependencies that cannot be updated in DEPENDENCY-NOTES.md.

### PR DESCRIPTION
**Pull request by mint-thompson**
**Description:**
Add eslint config file based on requirements for updated eslint dependencies.

Remove @types/tar dependency because types are now included in the tar dependency.

Minor usage updates for tar and commander based on changes to package export.

Minor fixes for linting rules.

**Testing Instructions:**
Confirm that all tests pass. Confirm that CLI works as expected. There will be additional review and testing when the `new-fpl` branch is being pulled into `main`.

**Related Issue:**
Fixes #27

### CODE REVIEW

File: package.json
Comment (jafeltra) [inline package.json:1]: I can't comment on the line itself, but for `@types/node`, even though we aren't going to update to the v22.x versions, I think we can update to the latest v20.x version.

**Review by jafeltra — COMMENTED** [review]
This looks good! I just left one small comment.

File: package.json
Comment (mint-thompson) [reply package.json:1]: Good idea! I've updated that package in the latest commit.

**Review by jafeltra — APPROVED** [review]
Looks good!

**Review by cmoesel — APPROVED** [review]
Thanks Mint!

**Pull request merged**